app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -274,6 +292,17 @@ private void showSnackBarWithRetry() {
                 R.string.achievements_fetch_failed, R.string.retry, view -> setAchievements());
     }
 
+    /**
+     * Shows a snack bar which tells the users that their wiki edit count is so high that it (in some cases)
+     * wreks havoc with the Achievements calculator due to which request may time out
+     * Well this is the Ultimate Achievement 
+     */
+    private void showSnackBarUltimateAchievementWithRetry() {
```

Comment:
nit: Making this just `showSnackBarWithRetry(int numberOfEdits)`, which would remove a lot of duplicate logic.